src/modules/tests/test-data/cat-adi-part2/vehicle-checks/vehicle-checks.cat-adi-part2.reducer.ts

```diff
@@ -13,6 +14,43 @@ export function vehicleChecksCatADI2Reducer(
   state: CatADI2UniqueTypes.VehicleChecks = initialState,
   action: vehicleChecksCatADI2ActionTypes.Types,
 ): CatADI2UniqueTypes.VehicleChecks {
+  let newShowMeQuestionsState = state.showMeQuestions;
```

Comment:
not sure if this logic should be in the reducer but might be wrong, think we should all discuss

Reply:
removed